powersimdata/data_access/fs_helper.py

```diff
@@ -1,9 +1,14 @@
+import logging
+
 import fs
 from fs.multifs import MultiFS
 
 from powersimdata.data_access.ssh_fs import WrapSSHFS
 from powersimdata.utility import server_setup
 
+logger = logging.getLogger("azure.core.pipeline.policies.http_logging_policy")
+logger.setLevel(logging.WARNING)
```

Comment:
What `logger.setLevel(logging.WARNING)` does? Just being curious.

Reply:
It filters out a bunch of output from the azure sdk, which I guess are at the `INFO` level (as compared to `WARNING`). So not actually helpful to a user, and honestly I don't know why they would have that output by default.. probably unintentional.